### PR TITLE
ex_31.py

### PR DESCRIPTION
**Pull request by amymhaddad**
When getting into a fitness program, you may want to figure out your target heart rate so you don’t overexert yourself. The Karvonen heart rate formula is one method you can use to determine your rate. Create a program that prompts for your age and your resting heart rate. Use the Karvonen for- mula to determine the target heart rate based on a range of intensities from 55% to 95%. Generate a table with the results as shown in the example output. 

The formula is
TargetHeartRate = (((220 − age) − restingHR) × intensity) + restingHR

### CODE REVIEW

File: ex_31.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@author: amyhaddad
+"""
+
+
+# Step 1: get input from user 
+age = int(input("Enter your age: "))
+resting_heart_rate = int(input("Enter your resting heart rate: "))
+
+# Step 2: write "for" loop to determine target heart rate based on given intensities (55% to 95%)
+for value in range(55, 100, 5):
```
Comment (paulghaddad) [inline ex_31.py]: Very impressive!

Comment (paulghaddad) [reply ex_31.py]: Consider making 55 and 100 constants, for whatever they represent.

File: ex_31.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    intensity_as_decimal = value / 100
+    target_hr = (((220 - age) - resting_heart_rate) * intensity_as_decimal) + resting_heart_rate
+    print(f"At {value}%, your heart rate should be {int(target_hr)}.")
```
Comment (paulghaddad) [inline ex_31.py]: Excellent!

**Comment by amymhaddad** [comment]
@paulghaddad great point on using constants with range! This exercise is updated.

File: ex_31.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Step 1: write constants
+LOW_INTENSITY = 55
```
Comment (paulghaddad) [inline ex_31.py]: Exactly

File: ex_31.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+HIGH_INTENSITY = 95
+INTERVAL = 5
+
+# Step 2: get input from user
```
[the remark this answers is not in the document]
Comment (paulghaddad) [reply ex_31.py]: Delete the trailing space here; same on comments below.

File: ex_31.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+INTERVAL = 5
+
+# Step 2: get input from user 
+age = int(input("Enter your age: "))
+resting_heart_rate = int(input("Enter your resting heart rate: "))
+
+# Step 3: write loop to determine target HR 
+for value in range(LOW_INTENSITY, HIGH_INTENSITY, INTERVAL):
+    intensity_as_decimal = value / 100
+    target_hr = (((220 - age) - resting_heart_rate) * intensity_as_decimal) + resting_heart_rate
```
[the remark this answers is not in the document]
Comment (paulghaddad) [reply ex_31.py]: Don't worry about this comment. I hate the line too long one myself.

**Comment by amymhaddad** [comment]
@paulghaddad I think I got the trailing white space.

**Pull request merged**